docs/hyperloop/notifications.md

```diff
@@ -0,0 +1,105 @@
+---
+sort: 3
+title: Notifications
+---
+
+## <a name="notifications"></a>Notifications
+* There are different types of notifications that the user receives, each related to a specific topic. A notification does not necessarly request an action to be taken, but informs the user aboout a specific element, and depending on the situation, the user can decide the possible next steps.
```

Comment:
Typo about

Reply:
In the top we can also explain that one remove a notification by clicking X or go to the window of notifications to see and remove all